[R2] Cleanup public bucket docs (#6002)
cloudflare/cloudflare-docs: content/r2/data-access/public-buckets.md

The domain is no longer connected to your bucket and will no longer appear in the connected domains list.

## Enable managed public access for your bucket

> Enabling managed public access for your bucket will make the content of your bucket available to the Internet through a subdomain on the Cloudflare managed http://r2.dev domain.

Reviewer: ```suggestion
Enabling managed public access for your bucket will make the content of your bucket available to the Internet through a `r2.dev` Cloudflare subdomain.
```

![Choose R2 from the side menu](/r2/static/public-buckets-r2.png)

2. In. **R2**, select the bucket you want to enable public access for.

![Select your bucket](/r2/static/public-buckets-bucket.png)

3. Go to **Settings**.

![Select settings](/r2/static/public-buckets-settings.png)

4. In **Settings**, go to **Bucket Access**.

![Select bucket access](/r2/static/public-buckets-not-allowed.png)

5. Under **Bucket Access**, select **Allow Access**.

![Allow access](/r2/static/public-buckets-allow-access.png)

6. You will be prompted to confirm your choice. In the confirmation dialog, type ‘allow’ to confirm and select **Allow**.
7. You can now access the bucket and its objects using the Public Bucket URL.

![Show Public Bucket URL](/r2/static/public-buckets-allowed.png)

You can review if your bucket is publicly accessible by going to your bucket and checking that **Public URL Access** states **Allowed**.

{{<Aside type="note">}}

To enable access management, Cache and bot management features, you must set up a custom domain when enabling public access to your bucket.

{{</Aside>}}

## Disable managed public access for your bucket

> Your bucket will not be exposed to the Internet as a subdomain on https://r2.dev after you disable public access.
> If you have connected other domains, the bucket will remain accessible on those domains.

Reviewer: ```suggestion
Your bucket will not be exposed to the Internet as an `r2.dev subdomain` after you disable public access. If you have connected other domains, the bucket will remain accessible on those domains.
```
